online-shop/src/css/Header.css

```diff
@@ -0,0 +1,29 @@
+.AppHeader {
+    background-color: #343a44;;
+    padding-bottom: 40px;
+}
+
+.HeaderTitle{
+    text-align: left;
+    font-weight: bolder;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-size: 1.5em;
+    margin-left: 5%;
+    color: crimson;
+}
+  
+.NavLink {
+    margin-left: 5%;
+    margin-top: 5%;
+    color: white;
+}
+
+nav {
```

Comment:
Evita folosirea tag-urilor de html in fisierele de css, ca o sa ai situatii cand un fisier iti suprascrie stilurile din alt fisier. Evita si id-urile pe cat posibil. Foloseste clase.